gridstatus/ercot.py

```diff
@@ -328,15 +328,13 @@ def get_fuel_mix(self, date, verbose=False):
 
         dfs = []
         for day in data["data"].keys():
-            df = (
-                pd.DataFrame(data["data"][day])
-                .applymap(
-                    lambda x: x["gen"],
-                    na_action="ignore",
+            df = pd.DataFrame(data["data"][day])
+            df_transformed = df.apply(
+                lambda col: col.apply(
+                    lambda x: x.get('gen') if isinstance(x, dict) else pd.NA
```

Comment:
The ruff linter is complaining this needs to be `"gen"`.

Reply:
🤦 ... I see you guys use black formatting 🎉 . Fixed the formatting with black on the latest commit!